docs/resource-reference.md

```diff
@@ -9925,7 +9925,7 @@ Ex. "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.<br/>
 
 
 Persistence defines the persistence configuration. If empty k0smotron
-will use emptyDir as a volume.
+will use emptyDir as a volume. More info about persistence [here](configuration.md#persistence).
```

Comment:
This file is autogenerated from the comments in `*_types.go` file:
https://github.com/k0sproject/k0smotron/blob/main/api/k0smotron.io/v1beta1/k0smotroncluster_types.go#L58

It needs to be updated there and then generated with `make docs-generate-reference`

Reply:
fixed 👍